showcases/react-showcase/src/components/index.tsx

```diff
@@ -1,51 +1,67 @@
-import { type CSSProperties, type ReactElement } from 'react';
-import { DBDivider } from '../../../../output/react/src';
+import { DBCodeDocs, DBDivider, DBLink } from '../../../../output/react/src';
+import useQuery from '../hooks/use-query';
+import {
+	type DefaultComponentProps,
+	type DefaultComponentVariants
+} from './data';
+import { useLocation } from 'react-router-dom';
 
-export type DefaultComponentExample = {
-	name: string;
-	example: ReactElement;
-	style?: CSSProperties;
-
-	className?: string;
-};
-
-export type DefaultComponentVariants = {
-	name: string;
-	examples: DefaultComponentExample[];
-};
-
-export type DefaultComponentProps = {
-	title: string;
-	description?: ReactElement;
-	variants: DefaultComponentVariants[];
-};
+const VariantList = ({ examples }: DefaultComponentVariants) => (
+	<DBCodeDocs
+		className="variants-card"
+		codeSnippets={examples
+			.filter((example) => example.code)
+			.map((example) => `/* ${example.name} */\n${example.code}`)}>
+		<dl className="variants-list">
+			{examples.map((example, exampleIndex) => (
+				<div
+					key={`${example.name}-${exampleIndex}`}
+					style={example.style}
+					className={example.className}>
+					<dt className="example-name">
+						<small>{example.name}</small>
```

Comment:
```suggestion
						{example.name}
```

Please use CSS for this styling purpose.